client/blocks/comments/post-comment-list.jsx

```diff
@@ -174,12 +174,12 @@ class PostCommentList extends React.Component {
 	 * @param {Number} take How many top level comments to take
 	 * @returns {Object} that has the displayed comments + total displayed count including children
 	 */
-	getDisplayedComments( commentIds, take ) {
+	getDisplayedComments( commentIds, numberToTake ) {
 		if ( ! commentIds ) {
 			return null;
 		}
 
-		const displayedComments = commentIds.take( take );
+		const displayedComments = take( commentIds, numberToTake );
```

Comment:
note also that this is often accomplished with `commentIds.slice( 0, numberToTake )`

I'm happy either way :smile: